module_prototyper/README.rst

```diff
@@ -0,0 +1,123 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Module Prototyper
+=================
+
+This module allows the administrator to prototype new features and export them as module. 
+Functional people can prepare the job for a developer who is left with the logic to implement 
+in addition to everything the prototype does not export yet.
+
+Installation
+============
+
+Make sure you have Jinja2 version 2.7.3 or higher installed::
+
+$ pip install --upgrade Jinja2==2.7.3
+
+Configuration
+=============
+
+No configuration required.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* install the dependencies of your future module
+* customize your instance by adding fields and creating inherited views
+* create your menu items and their window actions
+* prepare your data and demo data by creating filters
+* create your own groups with access rights and record rules
```

Comment:
These 2 steps are not yet covered by the module, so you should remove them from the usage until the module get support for them.